uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/fragment/CheckoutFragment.kt

```diff
@@ -36,6 +36,7 @@ import java.util.HashMap
 
 internal class CheckoutFragment : Fragment() {
     private lateinit var checkoutActionButton: LoadingButtonView
+    private lateinit var passengerActionButton: LoadingButtonView
```

Comment:
I don't really like the name. It's the main booking button that has a (sometimes visible) loading hud on it. I'm not saying to rename it right now if it would touch too much code, but we should at least document it for now and update the name later.

Reply:
Before, the passengerActionButton from the passengers page was named checkoutActionButton as the one on the checkout page, even though they were separate views. This lead to some weird bugs since listeners were attached on the ones from the checkout page and not on the passengers one

Reply:
Not sure about renaming it!?